_posts/2025-07-18-this-week-in-rails.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: A new touch option, accessing the editor from the error page and better migration logs
+categories: news
+author: claudiob
+og_image: assets/images/this-week-in-rails.png
+published: true
+date: 2025-07-18
+---
+
+
+Hi, it's [Claudio Baccigalupo](https://github.com/claudiob). So many great changes to the Rails codebase this week. Let’s check them out.
+
+
+[Add touch option to update_column(s)](https://github.com/rails/rails/pull/51455)  
+Particularly useful for ETL processes that rely on the `updated_at` timestamps instead of copying the whole table.
+
+[Add support to open files in the code editor from the crash page](https://github.com/rails/rails/pull/55295)  
+All the [most common editors](https://github.com/igorkasyanchuk/rails/blob/b22fd5a519f92e5c51c62e3b74b4407e492b89e2/activesupport/lib/active_support/editor.rb#L56-L68) are supported from Atom to Zed.
+
+[Fix errors when query string keys have invalid encoding](https://github.com/rails/rails/pull/55319)  
+Bots hitting your Rails app with malformed requests could generate noisy 500 Server Errors. After this PR the requests are killed without littering your logs.
```

Comment:
What is BadRequest? 400?
Technically it will still likely end up in some logs, but importantly won't be reported as an app exception.